data_collection/gazette/spiders/ba_alagoinhas.py

```diff
@@ -0,0 +1,7 @@
+from gazette.spiders.base import DoemGazetteSpider
```

Comment:
**issue (blocking)**: After https://github.com/okfn-brasil/querido-diario/pull/377 merged, the location of the base spider changed, so it will be necessary to update this import statement from all spiders that uses `DoemGazetteSpider`.

Reply:
Done